matrices.tex

```diff
@@ -25,6 +25,8 @@
 {x - sqrt(y*(x + y)): 1, x + sqrt(y*(x + y)): 1}
 \end{verbatim}
 
+%% TODO a Python list is not a dense representation...
+%% Is this supposed to say flat? - scopatz
```

Comment:
Also, here is another document that uses the term dense w.r.t. to contiguous memory https://www.kernel.org/doc/ols/2007/ols2007v1-pages-141-152.pdf